docs/deployment/getting_started_locally.md

```diff
@@ -142,6 +142,31 @@ Thus, leader election, health and readiness checks for `gardener-admission-contr
 
 If you have similar problems with other components which are not deployed by skaffold, you could temporarily turn off the leader election and disable liveness and readiness probes there too.
 
+### Debugging in GoLand
+
+1. Edit your **Run/Debug Configurations**.
+2. Add a new **Go Remote** configuration.
+3. Set the port to `56268` (or any increment of it when debugging multiple components).
+4. _Recommended:_ Change the behavior of **On disconnect** to **Leave it running**.
+
+### Debugging in VS Code
+
+1. Create or edit your `.vscode/launch.json` configuration.
+2. Add the following configuration:
+
+```json5
+{
+  "name": "go remote",
+  "type": "go",
+  "request": "attach",
+  "mode": "remote",
+  "port": 56268, // or any increment of it when debugging multiple components
+  "host": "127.0.0.1"
+}
+```
+
+Since the [ko](https://skaffold.dev/docs/builders/builder-types/ko/) builder is used in Skaffold to build the images it's not necessary to specify the `cwd` and `remotePath` options as they match the workspace folder ([ref](https://skaffold.dev/docs/workflows/debug/#skaffold-debug-using-the-vs-code-go-extension)).
```

Comment:
```suggestion
Since the [ko](https://skaffold.dev/docs/builders/builder-types/ko/) builder is used in Skaffold to build the images, it's not necessary to specify the `cwd` and `remotePath` options as they match the workspace folder ([ref](https://skaffold.dev/docs/workflows/debug/#skaffold-debug-using-the-vs-code-go-extension)).
```